config.toml

```diff
@@ -1,29 +1,106 @@
+# The global section has parameters that apply globally to the relayer operation.
+# https://hermes.informal.systems/config.html
```

Comment:
To make it easier to maintain, I suggest we no longer keep
the description of fields in
https://hermes.informal.systems/config.html.
Instead, we just present this file directly in https://hermes.informal.systems/config.html (with no additional discussion of the fields).

I added the label "needs guide update" to remember to update the guide correspondingly.

Reply:
I have now removed the whole Sections section.